models/Recipe.model.js

```diff
@@ -2,9 +2,34 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title: String,
```

Comment:
The title should also be required, we would like to know what the name of that fantastic italian dish :D